### ISSUE TITLE
Can not get configuration menu bar back

### ISSUE THREAD

**Issue by alexanderwwagner** [virt-manager/virt-manager#633]
**Distro**: Arch Linux 
**Virt-Manager version**: 4.1.0

I have created some VMs with virt-manager GUI.
Works great! But at one VM I do not get the configuration view back.
If I open the vm with "open" I get directly the graphical console and can not edit my configuration...
I guess I enabled any kind of fullscreen here? But I did some research and could not find any related key to get my required menu bar back

Missing menu bar to switch to configuration mode in vm1...:
![image](https://github.com/virt-manager/virt-manager/assets/45471644/4bdf3b6f-f8a3-4ac6-b1bc-6aed3081ad27)

I would like to have it look like this in vm2:
![image](https://github.com/virt-manager/virt-manager/assets/45471644/a036a9cd-0cc7-4440-a656-2eb34052b6a9)

**Comment by jatinsehrwt** [comment]
Have you tried maximizing the window, they might be hidden at top.

I guess you are using KDE, to get windows full screen use <super>+<pg-up> or <alt>+<f3>, then there is a setting to show/hide toolbar in "view -> toolbar", check if it is enabled.

**Comment by alexanderwwagner** [comment]
Yes I use KDE i can go to fullscreen with "F11" in each other app, browser...
But If I use the key on virtmanager nothing happens...
The "+" or "-" key does nothing on my setup...

**Comment by alexanderwwagner** [comment]
Where is the information "fullscreen" stored?
I can not find anything about it in the +.xml file of the affected VM.

**Comment by phrdina** [comment]
Hi, virt-manager stores the window details in [dconf](https://wiki.gnome.org/Projects/dconf), the path within dconf is `/org/virt-manager/virt-manager/vms/{UUID}` where `UUID` is the <uuid></uuid> element from VM XML without dashes.

You can run this `virsh dumpxml {VM} --xpath "/domain/uuid/text()" | sed "s/-//g"` with your VM name to figure out the UUID part for dconf.

**Comment by chewblacka** [comment]
I also ran into this same issue on Plasma6. No config menu bar.  It was happening on all machines. For me the following command fixed it (it force resets all dconf settings):

`dconf reset -f /org/virt-manager/virt-manager/`

**Comment by chewblacka** [comment]
Forgot to say, for me the problem arose on KDE because I had global menu enabled on my panel. It doesn't seem to work properly with virt-manager, although it does stop the menu from appearing. So you can't see the View menu to add / remove the toolbar.

**Comment by alexanderwwagner** [comment]
@phrdina 
Okay in the meantime i tried your described fix, with no success.
Now I tried this command
`dconf reset -f /org/virt-manager/virt-manager/`

My VMs are running on a remote server... and I connected to them with virt-manager gui from my developer notebook.
After I fired the command I am not longer able to connect to the remote VMs via virt-manager...

It seems my ssh keys are not longer working....
How can I solve this?
